root/usr/share/nethesis/NethServer/Template/TlsPolicy.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+echo $view->header()->setAttribute('template', $T('TlsPolicy_Description'));
+
+echo $view->panel()
+    ->insert($view->selector('policy', $view::SELECTOR_DROPDOWN))
+    ->insert($view->fieldset()->setAttribute('template', $T('ServiceExemption_label')))
+    ->insert($view->checkbox('HttpdAdminExeptionStatus','enabled')->setAttribute('uncheckedValue', 'disabled'))
```

Comment:
`exeption`: is it a typo?

Why this checkbox is needed? How is a "service exemption"  defined? How does it differ from the base policy?

Reply:
yes probably :D